69.md

```diff
@@ -0,0 +1,205 @@
+
+Intolerance & Hate (IH) is clearly in the eye of the beholder and it's likely that it will be ignored by the majority of relays. But filtering on intolerance and hate is necessary to facilitate the development of safe spaces on Nostr for vulnerable minority groups who are themselves often the target of intolerance and hate.
+
+Misinformation is an extremely touchy issue. If you notice there's no general MI code. Instead misinformation is only supported for "Your money or your life" situations - where serious harm can come to the user.
+
+It may seem having both "nudity & sex" (NS) and "pornography" (PN) as categories is redundant. Yes there is overlap between them, but "pornography" is more of a context while "nudity & sex" is more of a content type. Each is capable of giving relevant information the other can't give.
+
+Prostitution (IL-hkr) is only on the list because of FOSTA/SESTA - which is the one exception to CDA Section 230 and has to be taken seriously by all US-based relays.
+
+
+
+Examples:
+
+
+Self-reporting a post...
+{
+    "pubkey": "<pub-key>",
+    "created_at": 1000000000,
+    "kind": 1,
+    "tags": [
+      ["t", "hastag"],
+      ["content-warning", "Wet t-shirt contest", ""NS-ero"]
+    ],
+    "content": "(Racy content goes here)",
+    "id": "<event-id>"
+}
+
+3rd party reporting of same post...
+{
+  "kind": 1984,
+  "tags": [
+    [ "e", <eventId>, "NS-ero"],
+    [ "p", <pubkey>]
+  ],
+  "content": "Wet t-shirt contest",
+  ...
+}
+
+
+Self-reporting of an entire profile...
+{
+"id": "<event-id>"
+    "pubkey": "<pub-key>",
+    "created_at": 1000000000,
+    "kind": 0,
+    "tags": [
+      ["t", "hastag"],
+      ["content-warning", "Posts frequently feature drawings of naked people.", "NS-nud,FA"]
+    ],
+    "content": "{\"name\":\"OttawaLifeDrawing\",\"about\":\"Ottawa's newest life drawing classes!\"}",
+    ...
+}
+
+3rd party reporting of same profile...
+{
+  "kind": 1984,
+  "tags": [
+    [ "p", <pubkey>, "NS-nud,FA"]
+  ],
+  "content": "Lots of drawings of nude people",
+  ...
+}
+
+
+Self-reporting of particular parts of a profile...
+{
+"id": "<event-id>"
+    "pubkey": "<pub-key>",
+    "created_at": 1000000000,
+    "kind": 0,
+    "tags": [
+      ["t", "hastag"],
+      ["content-warning", "I'm a trans pornstar. My avatar is G-rated, but the header pic is a bit racy. My website link goes to a porn site and most of my posts are porn-related.", "PN-trn,PN-trn-website,NS-ero-banner"]
+    ],
+    "content": "{\"name\":\"PollyPornstar\",\"about\":\"Award-winning trans pornstar\",\"banner\":\"https://somesite.com/somepic.jpg\",\"website\":\"https://pornsite.com\",\"picture\":\"https://somesite.com/anotherpic.jpg\"}",
+    ...
+}
+
+3rd party reporting of the same parts of the same profile...
+{
+  "kind": 1984,
+  "tags": [
+    [ "p", <pubkey>, "PN-trn,PN-trn-website,NS-ero-banner"]
+  ],
+  "content": "Trans pornstar with racy banner and website link to a porn site.",
+  ...
+}
+
```

Comment:
````suggestion
Self-reporting a post:

```
{
  "pubkey": <pubkey>,
  "kind": 1,
  "tags": [
    ["t", "hashtag"],
    ["content-warning", "Wet t-shirt contest", "NS-ero"]
  ],
  "content": "(Racy content goes here)",
  "id": <eventId>
}
```

3rd party reporting of same post:

```
{
  "kind": 1984,
  "tags": [
    [ "e", <eventId>, "NS-ero"],
    [ "p", <pubkey>]
  ],
  "content": "Wet t-shirt contest",
  ...
}
```


Self-reporting of an entire profile:

```
{
  "id": <eventId>,
  "pubkey": <pubkey>,
  "created_at": 1000000000,
  "kind": 0,
  "tags": [
    ["t", "hashtag"],
    ["content-warning", "Posts frequently feature drawings of naked people.", "NS-nud,FA"]
  ],
  "content": "{\"name\":\"OttawaLifeDrawing\",\"about\":\"Ottawa's newest life drawing classes!\"}",
  …
}
```

3rd party reporting of same profile:

```
{
  "kind": 1984,
  "tags": [
    [ "p", <pubkey>, "NS-nud,FA"]
  ],
  "content": "Lots of drawings of nude people",
  …
}
```

Self-reporting of particular parts of a profile:

```
{
  "id": <eventId>,
  "pubkey": <pubkey>,
  "created_at": 1000000000,
  "kind": 0,
  "tags": [
    ["t", "hastag"],
    ["content-warning", "I'm a trans pornstar. My avatar is G-rated, but the header pic is a bit racy. My website link goes to a porn site and most of my posts are porn-related.", "PN-trn,PN-trn-website,NS-ero-banner"]
  ],
  "content": "{\"name\":\"PollyPornstar\",\"about\":\"Award-winning trans pornstar\",\"banner\":\"https://somesite.com/somepic.jpg\",\"website\":\"https://pornsite.com\",\"picture\":\"https://somesite.com/anotherpic.jpg\"}",
  …
}
```

3rd party reporting of the same parts of the same profile:

```
{
  "kind": 1984,
  "tags": [
    [ "p", <pubkey>, "PN-trn,PN-trn-website,NS-ero-banner"]
  ],
  "content": "Trans pornstar with racy banner and website link to a porn site.",
  ...
}
```
````